### ISSUE TITLE
Get all data for propID without having to input dates

### ISSUE THREAD

**Issue by bhilbert4** [spacetelescope/jwst_mast_query#43]
It would be nice if a user wanted to download all data for a given program, that they could do that without having to specify a lookbacktime or date_select value.

e.g., it would be nice if these worked: (they currently don't because lookbacktime defaults to 1 day, which is a good thing)
jwst_download.py -i nircam --propID 1068
jwst_download.py -i nircam --propID 1068 --obsnums 3

This can be done within the current framework. I think it would require an initial query to MAST to retrieve the dates that the data for the propID were acquired. And then use those dates as date_select values.

**Comment by juliengirard** [comment]
I agree, the default should be no lookbacktime (or lookbacktime > 30 years)
